src/renderers/dom/shared/DOMProperty.js

```diff
@@ -253,6 +238,14 @@ var DOMProperty = {
     }
   },
 
+  getPropertyInfo(name) {
+    var lowerCased = name.toLowerCase();
+
+    return DOMProperty.properties.hasOwnProperty(lowerCased)
+      ? DOMProperty.properties[lowerCased]
+      : null;
+  },
```

Comment:
Just a final word here. In this scenario, all attributes are referenced as their lowercase form internally. I've hidden that within `getPropertyInfo` to avoid needing to know about lowercasing attributes outside of `DOMProperty.js`.